eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/HTTPClientHandler.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.eventmesh.runtime.boot.EventMeshHTTPServer;
 import org.apache.eventmesh.runtime.common.EventHttpHandler;
 import org.apache.eventmesh.runtime.core.protocol.http.processor.inf.Client;
+import org.apache.eventmesh.common.Constants;
```

Comment:
The import should be placed here:
![image](https://user-images.githubusercontent.com/91315508/234749632-1661207d-4579-4377-871f-bee6e81baa74.png)